hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/x509/keys/TestKeyStorage.java

```diff
@@ -208,6 +209,16 @@ public void testReadKeyFailToWrite() throws Exception {
       assertThat(e.getMessage()).isEqualTo(ERROR_MSG);
     }
 
+    @Test
+    @DisplayName("an attempt to overwrite an internal key throws FileAlreadyExists exception.")
```

Comment:
Is this a typo?  testExternalKeysAreNotOverWritable -> testInternalKeysAreNotOverWritable?